tools/SourceKit/lib/SwiftLang/SwiftEditor.cpp

```diff
@@ -308,7 +309,8 @@ SwiftEditorDocumentFileMap::findByPath(StringRef FilePath, bool IsRealpath) {
 
   std::string Scratch;
   if (!IsRealpath) {
-    Scratch = SwiftLangSupport::resolvePathSymlinks(FilePath);
+    Scratch = resolveSymbolicLinks(FilePath,
+        llvm::vfs::getRealFileSystem().get());
```

Comment:
I haven't checked, how hard is it to pass through the FileSystem here? I suspect most callers should have one already (or have one accessible through eg. CompilerInstance).

Reply:
I updated the PR to use the ambient FileSystem.